src/app/content/components/Sidebar/styled/index.tsx

```diff
@@ -131,7 +105,7 @@ export const NavOl = styled.ol<{section: ArchiveTree}>`
   }}
 `;
 
-type DetailsComponentProps = {defaultOpen: boolean} & React.HTMLProps<HTMLDetailsElement>;
+interface DetailsComponentProps {defaultOpen: boolean; open: boolean; }
```

Comment:
It seems like the definition of `DetailsComponent` is generic enough to apply to the nav bar _and_ the attribution. Is that not the case?